helm/templates/querier-statefulset.yaml

```diff
@@ -60,11 +65,6 @@ spec:
               secretKeyRef:
                 name: {{ $secret.name }}
                 key: {{ $key }}
-          - name: HOSTNAME
-            valueFrom:
-              fieldRef:
-                apiVersion: v1
-                fieldPath: metadata.name
```

Comment:
This is needed, why remove this?

Reply:
https://github.com/AdheipSingh/parseable/blob/aks-helm/helm/templates/querier-statefulset.yaml#L50
This is present just refactored it

Reply:
oh right, my bad.